### DISCUSSION TITLE
Battery Cycle counts - Sunsynk batteries

### DISCUSSION THREAD

**Discussion by amctear** [kellerza/sunsynk#147] [Q&A]
Is it possible to have a sensor for each battery to provide cycle counts ?

I've created the custom sensors as below with Multi running perfectly. No values however ?

SENSORS += Sensor(603, "Bat1 SOC", "%"),
SENSORS += Sensor(617, "Bat1 Cycle"),
SENSORS += Sensor(625, "Bat2 SOC", "%"),
SENSORS += Sensor(611, "Bat2 Cycle"),
SENSORS += Sensor(631, "Bat3 SOC", "%"),
SENSORS += Sensor(639, "Bat3 Cycle"),
SENSORS += Sensor(645, "Bat4 SOC", "%"),
SENSORS += Sensor(653, "Bat4 Cycle"),
SENSORS += Sensor(659, "Bat5 SOC", "%"),
SENSORS += Sensor(667, "Bat5 Cycle"),
SENSORS += Sensor(673, "Bat6 SOC", "%"),
SENSORS += Sensor(681, "Bat6 Cycle"),

Alan

**Comment by kellerza** [comment]
I doubt this is supported by the Sunsynk (at least not when I tried it)

Alternatives would be to access your BMS via RS485 - For Hubble AM-2 someone had a suggestion - https://github.com/kellerza/sunsynk/discussions/135
